cli/src/plz/cli/configuration.py

```diff
@@ -157,3 +157,6 @@ def __getattr__(self, name):
             return self.data.get(name, self.properties[name].default)
         else:
             raise KeyError(name)
+
+    def as_dict(self):
+        return {name: self.__getattr__(name) for name in self.properties}
```

Comment:
Probably should use `getattr(self, name)`? I feel like you're not supposed to call `__getattr__` directly. Maybe I'm being paranoid though.

Reply:
Me being sleepy